src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -599,6 +605,9 @@ void drawing_classifier::train(gl_sframe data, std::string target_column_name,
     }
   }
 
+  state_update["num_examples"] = data.size();
```

Comment:
`num_of_training_example` might be more clear? To me, it's a bit ambiguous since it doesn't tell me whether it includes the validation set of not.

Reply:
Sadly we have used `num_examples` everywhere :(. We have to change it everywhere if we do it here.